test/dynamo/test_base_hop.py

```diff
@@ -209,9 +209,9 @@ def f(x, y):
             normalize_graph(bk.graphs[0]),
             """\
 class GraphModule(torch.nn.Module):
-    def forward(self, L_x_: "f32[3, 3]", L_y_: "f32[3, 3]"):
-        l_x_ = L_x_
+    def forward(self, L_y_: "f32[3, 3]", L_x_: "f32[3, 3]"):
```

Comment:
Inputs got reversed here, is that still expected?

Reply:
Found the issue pushing a PR before this one in a new stack